### PR TITLE
[New] How to Install and Configure FastCGI and PHP-FPM

### PR DESCRIPTION
**Pull request by leslitagordita**
* How to Install and Configure FastCGI and PHP-FPM on Ubuntu 18.04
* How to Install and Configure FastCGI and PHP-FPM on Debian 10
* How to Install and Configure FastCGI and PHP-FPM on CentOS 8

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 79fe419e1de869f68c904a75403eca2aa668ff71

https://deploy-preview-3164--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit fc4d8955a3956576940514f741ac92a797f0edec

https://deploy-preview-3164--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/web-servers/apache/how-to-install-and-configure-fastcgi-and-php-fpm-on-centos-8/index.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+1. Complete the steps in the [How to Install a LAMP Stack on CentOS 8](/docs/web-servers/lamp/how-to-install-a-lamp-stack-on-centos-8/) guide. After completing the LAMP stack guide, you should have an Apache virtual hosts configuration for your own website. This guide will continue to refer to the site as `example.com`.
+
+    {{< note >}}
+This guide's examples will use PHP version 7.3. When running commands related to PHP, ensure you replace any version numbers with your own system's PHP version.
+    {{</ note >}}
+
+## Install mod_fcgid and PHP-FPM
+
+In this section, you will install the mod_fcgid and PHP-FPM modules on your CentOS 8 Linode.
```
[the remark this answers is not in the document]
Comment (andystevensname) [reply docs/web-servers/apache/how-to-install-and-configure-fastcgi-and-php-fpm-on-centos-8/index.md]: Actually, upon further thought, there may be cases where these two packages might not be in backticks. Maybe when discussing the package they aren't in backticks, but when talking about the command they are? Use best judgement and make sure whatever the rational it's consistent throughout the guide.

File: docs/web-servers/apache/how-to-install-and-configure-fastcgi-and-php-fpm-on-centos-8/index.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  name: Linode
+title: "How to Install and Configure FastCGI and PHP-FPM on CentOS 8"
+h1_title: "Install and Configure FastCGI and PHP-FPM on CentOS 8"
+contributor:
+  name: Linode
+---
+
+`mod_fcgid` is an [Apache module](https://httpd.apache.org/mod_fcgid/) that uses the [FastCGI](https://en.wikipedia.org/wiki/FastCGI) protocol to provide an interface between Apache and Common Gateway Interface (CGI) programs. CGI helps a web server handle dynamic content generation and processing for scripting languages like PHP. This dynamic functionality is commonly used when running content management systems like WordPress on a LAMP stack.
+
+This guide will show you how to install `mod_fcgid` and `PHP-FPM` on CentOS 8. It will also provide a basic configuration that uses socket based connections, instead of TCP. These steps will enable you to run PHP through `mod_fcgid`. Running PHP through `mod_fcgid` helps to reduce the amount of system resources used by forcing the web server to act as a proxy and only pass files ending with the *php* file extension to PHP-FPM. Additionally, using PHP-FPM allows each virtual host to be configured to run PHP code as individual users.
```
Comment (andystevensname) [inline docs/web-servers/apache/how-to-install-and-configure-fastcgi-and-php-fpm-on-centos-8/index.md]: Need a period before *.php* file extension

File: docs/web-servers/apache/how-to-install-and-configure-fastcgi-and-php-fpm-on-debian-10/index.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+  name: Linode
+title: "How to Install and Configure FastCGI and PHP-FPM on Debian 10"
+h1_title: "Install and Configure FastCGI and PHP-FPM on Debian 10"
+contributor:
+  name: Linode
+---
+
+`mod_fcgid` is an [Apache module](https://httpd.apache.org/mod_fcgid/) that uses the [FastCGI](https://en.wikipedia.org/wiki/FastCGI) protocol to provide an interface between Apache and Common Gateway Interface (CGI) programs. CGI helps a web server handle dynamic content generation and processing for scripting languages like PHP. This dynamic functionality is commonly used when running content management systems like WordPress on a LAMP stack.
+
+This guide will show you how to install `mod_fcgid` and `PHP-FPM` on Debian 10. It will also provide a basic configuration that uses socket based connections, instead of TCP. These steps will enable you to run PHP through `mod_fcgid`. Running PHP through `mod_fcgid` helps to reduce the amount of system resources used by forcing the web server to act as a proxy and only pass files ending with the *php* file extension to PHP-FPM. Additionally, using PHP-FPM allows each virtual host to be configured to run PHP code as individual users.
```
Comment (andystevensname) [inline docs/web-servers/apache/how-to-install-and-configure-fastcgi-and-php-fpm-on-debian-10/index.md]: Need a period before *.php* file extension

File: docs/web-servers/apache/how-to-install-and-configure-fastcgi-and-php-fpm-on-ubuntu-18-04/index.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+  name: Linode
+title: "How to Install and Configure FastCGI and PHP-FPM on Ubuntu 18.04"
+h1_title: "Install and Configure FastCGI and PHP-FPM on Ubuntu 18.04"
+contributor:
+  name: Linode
+---
+
+`mod_fcgid` is an [Apache module](https://httpd.apache.org/mod_fcgid/) that uses the [FastCGI](https://en.wikipedia.org/wiki/FastCGI) protocol to provide an interface between Apache and Common Gateway Interface (CGI) programs. CGI helps a web server handle dynamic content generation and processing for scripting languages like PHP. This dynamic functionality is commonly used when running content management systems like WordPress on a LAMP stack.
+
+This guide will show you how to install `mod_fcgid` and `PHP-FPM` on Ubuntu 18.04. It will also provide a basic configuration that uses socket based connections, instead of TCP. These steps will enable you to run PHP through `mod_fcgid`. Running PHP through `mod_fcgid` helps to reduce the amount of system resources used by forcing the web server to act as a proxy and only pass files ending with the *php* file extension to PHP-FPM. Additionally, using PHP-FPM allows each virtual host to be configured to run PHP code as individual users.
```
Comment (andystevensname) [inline docs/web-servers/apache/how-to-install-and-configure-fastcgi-and-php-fpm-on-ubuntu-18-04/index.md]: Need a period before *.php* file extension

**Pull request merged**